src/components/footer.mjs

```diff
@@ -0,0 +1,16 @@
+import { html } from "../util/html.mjs";
+
+export function renderFooter() {
+	return html`
+		<footer>
+			<p>Want to contribute? Checkout the <a href="https://github.com/csstools/cssdb">cssdb on GitHub</a>.</p>
+			<p>Logos (except for Internet Explorer, BlackBerry, Baidu and QQ) are from <a href="https://github.com/alrra/browser-logos">alrra/browser-logos</a>. Check them out!</p>
```

Comment:
Should we update the disclaimer to include the oculus one?

Reply:
Yes! added this :)